lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
thanks @DFurnes and @weerd ! Sorry for my confusion on my end but are the steps to take to also send back the NS avatar picture in the [proxy route](https://github.com/DoSomething/northstar/blob/dev/app/Http/Controllers/SignupController.php#L32-L58) and to call this in [Signup.php's `build` function](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_signup/includes/Signup.php#L88)?

@weerd the avatar picture is confusing! It's the avatar picture on the mobile app that gets saved to aws. As I understand, on the site, the only people who have profile pictures are admins? cc @aaronschachter in case I got any of that incorrect.